setup.py

```diff
@@ -430,32 +403,35 @@ def is_Numpy_installed():
     ]
 
 # Add extensions that requires NumPy to build
-if is_Numpy_installed():
+try:
     import numpy
-    numpy_include_dir = numpy.get_include()
-    EXTENSIONS.append(
-        Extension('Bio.Cluster.cluster',
-                  ['Bio/Cluster/clustermodule.c',
-                   'Bio/Cluster/cluster.c'],
-                  include_dirs=[numpy_include_dir],
-                  ))
-    EXTENSIONS.append(
-        Extension('Bio.KDTree._CKDTree',
-                  ["Bio/KDTree/KDTree.c",
-                   "Bio/KDTree/KDTreemodule.c"],
-                  include_dirs=[numpy_include_dir],
-                  ))
-    EXTENSIONS.append(
-        Extension('Bio.motifs._pwm',
-                  ["Bio/motifs/_pwm.c"],
-                  include_dirs=[numpy_include_dir],
-                  ))
-    EXTENSIONS.append(
-        Extension('Bio.PDB.QCPSuperimposer.qcprotmodule',
-                  ["Bio/PDB/QCPSuperimposer/qcprotmodule.c"],
-                  include_dirs=[numpy_include_dir],
-                  ))
-
+except ImportError as e:
+    subprocess.call(["pip", "install", "numpy"])
```

Comment:
This will also fail if pip has an alias. It might be better to import pip.

    try:
        import pip
    except ImportError:
        print("pip is not found. Install numpy manually")
    else:
        pip.main(["install", "numpy"])